test/index.js

```diff
@@ -6,11 +6,13 @@ var expect = require('unexpected')
 var chanceGenerators = require('chance-generators');
 var tetons = fs.readFileSync(__dirname + '/data/tetons.exif');
 var IMG_0774 = fs.readFileSync(__dirname + '/data/IMG_0774.exif');
+var non_ascii = fs.readFileSync(__dirname + '/data/non-ascii.exif');
 
 describe('exif-reader', function() {
   it('should read tiff and exif data', function() {
     expect(exif(tetons), 'to equal',
-      { image:
+      { bigEndian: true,
```

Comment:
Thank you for the update. Please can you document the new `bigEndian` top-level property in the README then we're good to merge.

Reply:
Ok, I updated it accordingly.